Romina.Api/Startup.cs

```diff
@@ -3,14 +3,21 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Romina.Api.Repositories;
 
 namespace Romina.Api
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
+        public Startup(IHostingEnvironment env)
         {
-            Configuration = configuration;
+            var builder = new ConfigurationBuilder()
```

Comment:
hey bud, you wont need to make this change here as the new version of net core does this heavier lifting in the `Program.cs`

so you would want to revert any changes here in the constructor